browser/ui/BUILD.gn

```diff
@@ -339,6 +337,21 @@ source_set("ui") {
     ]
   }
 
+  # brave's obsolete infobar is only used on Windows now.
+  if (is_win) {
+    sources += [
+      "startup/brave_obsolete_system_infobar_delegate.cc",
+      "startup/brave_obsolete_system_infobar_delegate.h",
+    ]
+
+    if (toolkit_views) {
```

Comment:
what's the reasons behind this condition? is it necessary?

Reply:
It's used because below two files use toolkit views api.
Although It'll work w/o `toolkit_views` because `toolkit_views` is enabled on Windows,
I think all views implementation is in `toolkit_views` guard in chromium.